data-service/src/main/java/com/nova/dataservice/controller/LabReportLocation.java

```diff
@@ -34,7 +38,21 @@ public ResponseEntity<String> uploadLabReportLocation(@RequestParam("file") Mult
 			return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body("Failed to upload the file.");
 		}
 	}
+	@GetMapping(value = "getAllReportsByAppointmentI")
```

Comment:
proper naming should be used -- getAllReportsByAppointmentI replace with "getAllReportsByAppointmentId"